pages/mercury_tour_page.py

```diff
@@ -0,0 +1,76 @@
+import time
+from selenium.webdriver.support.select import Select
+
+
+class BasePage(object):
+
+    def __init__(self, driver):
+        # instantiate driver
+        self.driver = driver
+
+
+class MainWelcomePage(BasePage):
+
+    def title_matches(self):
+        # Find out Header
+        return "Welcome: Mercury Tours" in self.driver.title
+
+    def login_cred(self):
+        # Give login cred and click submit
+        user_name = self.driver.find_element_by_name('userName')
+        user_name.send_keys("admin")
+        password_keys = self.driver.find_element_by_name('password')
+        password_keys.send_keys("admin")
+        submit = self.driver.find_element_by_name('submit')
+        submit.click()
+
+    def flights_vertical(self):
+        # Move to flights vertical
+        self.driver.find_element_by_link_text('Flights').click()
+
+    def pass_count(self):
+        passenger = self.driver.find_element_by_name('passCount')
+        Select(passenger).select_by_visible_text("4")
+
+    def from_port(self):
+        departing_port = self.driver.find_element_by_name('fromPort')
+        Select(departing_port).select_by_visible_text("Frankfurt")
```

Comment:
Same is the case for every function
``` 
def from_port(self, port):
    departing_port = self.driver.find_element_by_name('fromPort')
    Select(departing_port).select_by_visible_text(port)
```
When you will call this function: 
```
from_port('Frankfurt')
```

Reply:
Done